liquibase-core/src/main/java/liquibase/parser/core/xml/XMLChangeLogSAXParser.java

```diff
@@ -150,6 +151,15 @@ public void fatalError(SAXParseException exception) throws SAXException {
             }
 
             throw new ChangeLogParseException("Invalid Migration File: " + reason, e);
+        } catch (InvalidPathException e) {
+            if (e.getInput().startsWith("s3://")) {
+                final String message = "ERROR: The file '" + e.getInput() + "' was not found. " +
+                        "The <changelog|snapshot> file property cannot be configured with " +
+                        "a fully qualified path, but must be a relative path on the property, " +
+                        "and any local or remote base of the path set on the '--search-path' property.";
+                throw new ChangeLogParseException(message);
```

Comment:
Make sure you either log or include the original exception `e` with the new exception you are throwing. Log would probably be best.

Reply:
done!